src/cli.ts

```diff
@@ -141,8 +149,10 @@ body {
 `;
   if (args.output) {
     fs.writeFileSync(args.output, output, {encoding: 'utf-8'});
-  } else {
+  } else if (!process.stdout.isTTY) {
```

Comment:
I think this is better expressed as some sort of flag, because otherwise you run into weird behaviors where like `foo | cat` has different behavior than `foo`.

How about: `-o somefile.html` writes to a file, otherwise always open in browser?

Reply:
I can do that, but what would happen in your ssh session? Would it just fail? The `foo | cat` vs. `foo` behavior actually makes sense to me. You see this all the time, e.g. with [jq](https://stedolan.github.io/jq/), `jq . foo.json` pretty-prints a JSON file with color to your terminal, but `jq . foo.json | cat` drops the color unless you specifically ask for it.